docs/extend/extend-apiml/redirect.md

```diff
@@ -0,0 +1,34 @@
+# Handling redirect by the API Gateway
+
+When a service routed through the API Gateway responds with a redirect status code (3xx) and a URL in the Location header, the API Gateway modifies the header to ensure that the client can properly access the new URL.
+
+:::note
+No steps are required from the API service developer as the redirect is performed auomatically by the API Gateway.
+:::
+
+## How Location Header is handled by the API Gateway
```

Comment:
It appears that there are no configuration steps that need to be performed by the API service developer. Is this article strictly informational? Is there option to disable this feature of the Gateway?